pom.xml

```diff
@@ -63,6 +63,8 @@ limitations under the License.
     <slf4j.version>1.7.25</slf4j.version>
     <commons-logging.version>1.2</commons-logging.version>
     <jsr305.version>3.0.2</jsr305.version>
+    <log4j2.version>2.15.0</log4j2.version>
```

Comment:
you can probably clean up this same property in the tools module and reference this instead.

Reply:
done!